SWIPs/swip-message_to_honey.md

```diff
@@ -48,19 +53,19 @@ This section describes the interaction between the nodes and the oracle in more
   }
 }
 ```
-* After `TTL` expires, nodes will query the price oracle for a new honeyPrices object. 
+* After `TTL` expires, nodes will query the price oracle for a new `honeyPrices` object. 
 * The `TTL` is set as a state-variable in the smart-contract and can be updated.
-* To ensure that it is not possible that a new entry in the `prices` array becomes valid, while not all nodes are updated, the smart-contract or update-policy must ensure that the `validFrom` of new prices must be at least `TTL` seconds in the future.
-* Upon start-up, nodes will look at the contents of their local database and query the price oracle if no `honeyPrices` object is found, or the `TTL` of their cached `honeyPrices` object has expired. 
+* To ensure that it is not possible that a new entry in the `prices` array becomes valid before all nodes are updated, the smart-contract or update-policy must ensure that the `validFrom` of new prices must be at least `TTL` seconds in the future.
+* Upon start-up, nodes will look at the contents of their local cache and query the price oracle if no `honeyPrices` object is found, or the `TTL` of their cached `honeyPrices` object has expired. 
 * If the oracle cannot be reached when the `TTL` of the last `honeyPrices` object is expired, nodes will continue to apply the `prices` as instructed by the expired `honeyObject` and will start to query the oracle at a more regular interval to re-establish connection. 
```

Comment:
what about the edge case that a fresh node starts up (thus has no prices) and can not reach the oracle?

Reply:
Interesting. Maybe there could be a mechanism where you can ask your neighbours? It is true that adding this oracle now means everyone has to maintain connection with Ethereum node.